src/main/java/mekanism/common/block/BlockBasic.java

```diff
@@ -497,6 +508,22 @@ public boolean isFullCube(IBlockState state) {
         return type != null && type.isFullBlock;
     }
 
+    @Override
+    @Deprecated
+    public boolean isFullBlock(IBlockState state) {
+        BasicBlockType type = BasicBlockType.get(state);
+        return type != null && type.isFullBlock;
+    }
+
+    @Override
+    public int getLightOpacity(IBlockState state, IBlockAccess world, BlockPos pos) {
+        BasicBlockType type = BasicBlockType.get(state);
+        if(type != null) {
+            return type.isOpaqueCube ? 255 : 0;
+        }
+        return 0;
```

Comment:
To be similar to how the other ones are, you want you could make this just be
```java
return type != null && type.isOpaqueCube ? 255 : 0;
```
Up to you though given it doesn't really matter, but once a decision is made and depending on if you are editing other things anyways you may want to.

Reply:
btw i changed this and implemented the own Values for Osmium Blocks a few days ago and i will upload it until i got told what i should set the BlockBasicStandard Resistance to.